.gitignore

```diff
@@ -9,6 +9,9 @@
 .project
 .settings/**
 
+# Vscode files
+.vscode/**
+
```

Comment:
slightly orthogonal to the PR, but I wanted to give *this - 

https://marketplace.visualstudio.com/items/lukehoban.Go 
 
a try for a while.  I can remove if desired.